### DISCUSSION TITLE
Want suggestion to contribute

### DISCUSSION THREAD

**Discussion by ajay-k07** [serverpod/serverpod#429] [General]
Hi I have been working on flutter for past 2 years.and now i feel like i should start contributing to the community and i feel like I can do that in serverpod which will be really helpful for most of the people. Please let me know how can I do small small task and then let's me take something gud one. Since I'm new to open source contribution i believe starting from small small task will be easy for me

**Comment by tusharbhambere** [comment]
Your can Start from Documentation

For example Authentication Part in documentation Just Implement Step by step if it's working then try to improve that part documentation Writte and improve steps for other reader from your experience
Else not working part you can fix that bug Update though PR also Update Documentation

Its Very Great Initiative 👏

**Comment by vlidholt** [comment]
Yes, the authentication part is where we need the most help right now. Both in the implementation, supporting more platforms and documentation. :)

**Reply by ajay-k07** [reply]
Sure, I have started working on that

**Reply by AndryHTC** [reply]
> Sure, I have started working on that 

Amazing... There is at least an entire team waiting for you 🤣.

Thank you for the effort

**Reply by ajay-k07** [reply]
@vlidholt can i get any reference project which has implemented authentication. i have completed documentation till modules setup .im trying to create email auth with example project for that can i get any reference project

**Reply by vlidholt** [reply]
I've just done some major improvements on the docs for authentication. You can find a preview here:
https://github.com/serverpod/serverpod_docs/blob/main/docs/04-concepts/07-authentication.md

What we need most help with is adding new methods for signing in. E.g. Facebook, Github. They would need to follow the same pattern as for the Google and Apple sign-ins in the auth module.

**Comment by ajay-k07** [comment]
hi guys finally completed my tutorial for email auth  please let me know if this can be added in the documentation [Email Auth](https://youtu.be/8dDucblphek) Next let me work on Google

**Reply by vlidholt** [reply]
That is very cool! I will have a look at it. :)

**Reply by ajay-k07** [reply]
Do you have any suggestions @vlidholt ?

**Comment by ajay-k07** [comment]
New Video for Google Sign In with Serverpod [Google SignIn With Serverpod](https://youtu.be/y7oLNlbWbUY)

**Reply by II11II** [reply]
Hi Mate, Can you make the video of how to connect to other clients (such as Postman, cli, or any js clients)?
I believe it will be helpful for others

**Reply by vlidholt** [reply]
Very cool, I will add these videos to the tutorial page in the documentation!

**Reply by ajay-k07** [reply]
Hey @II11II  Excited to share my latest tutorial 📹 on using Postman to test APIs created with Serverpod! Learn how to simplify your API testing process and streamline your workflow 💻 Check out the full video here: [https://youtu.be/339u_nukJoU](https://youtu.be/339u_nukJoU)

**Comment by ajay-k07** [comment]
Hi all added a new video for serverpod and s3

https://youtu.be/QD5J8YvQPPM

**Reply by adamantoise** [reply]
Why am I getting notification emails about this?  I did not subscribe to this thread and want nothing to do with it.

**Reply by AndryHTC** [reply]
> Why am I getting notification emails about this?  I did not subscribe to this thread and want nothing to do with it.

@adamantoise please keep a respectful tone while you try to solve your issue.

Could be that you subscribed to the entire Serverpod organization/repo...

Keep your smile e don't forget to contribute if you can 😄

**Reply by ajay-k07** [reply]
Hi @adamantoise sry about that , as @AndryHTC said please check your subscription 😄

**Reply by Quppa** [reply]
There was a spam comment on this thread (since deleted) that included 50 seemingly random GitHub usernames, including mine and @adamantoise's. I've never experienced this before, but I think GitHub should handle it better and remove subscriptions automatically in this scenario.

Edit: another report of this: https://github.com/community/community/discussions/42810

![image](https://user-images.githubusercontent.com/2142963/212469495-c574cd17-e32c-4538-82b7-27bfc5e3c8a4.png)

**Comment by ajay-k07** [comment]
Excited to share my latest tutorial 📹 on using Postman to test APIs created with Serverpod! Learn how to simplify your API testing process and streamline your workflow 💻 Check out the full video here: https://youtu.be/339u_nukJoU

**Reply by vlidholt** [reply]
If you tweet about it, I will retweet from the official ServerpodDev account!

**Reply by ajay-k07** [reply]
> If you tweet about it, I will retweet from the official ServerpodDev account!

I did a tweet 😃
